WalletWasabi.Tests/RegressionTests/SendTests.cs

```diff
@@ -99,7 +99,7 @@ public async Task SendTestsAsync()
 					waitCount++;
 					if (waitCount >= 21)
 					{
-						Logger.LogInfo("Funding transaction to the wallet did not arrive.");
+						Logger.LogInfo($"Funding transaction to the wallet {wallet.WalletName} did not arrive.");
```

Comment:
For consistency:

```suggestion
						Logger.LogInfo($"Funding transaction to the wallet '{wallet.WalletName}' did not arrive.");
```

?